src/Example/UserRegistrationBundle/Transfer/UserTransfer.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/*
+ * Copyright (c) 2012-2014 KUBO Atsuhiro <kubo@iteman.jp>,
+ * All rights reserved.
+ *
+ * This file is part of PHPMentors_Training_Example_Symfony.
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the BSD 2-Clause License which accompanies this
+ * distribution, and is available at http://opensource.org/licenses/BSD-2-Clause
+ */
+
+namespace Example\UserRegistrationBundle\Transfer;
+
+use Example\UserRegistrationBundle\Entity\User;
+
+class UserTransfer
```

Comment:
__[development][phpmd]__
* The property $ACTIVATION_EMAIL_ACTIVATION_URI is not named in camelCase.
* The property $ACTIVATION_EMAIL_FROM is not named in camelCase.
* The property $ACTIVATION_EMAIL_SUBJECT is not named in camelCase.
* The property $ACTIVATION_EMAIL_TEMPLATE is not named in camelCase.